src/renderer/inc/RenderEngineBase.hpp

```diff
@@ -21,39 +21,91 @@ Author(s):
 #pragma once
 namespace Microsoft::Console::Render
 {
+    struct BufferLineRenderData

```

Comment:
I think this is pretty badly organized and I don't have much experience in packing structs. It does not seem to cause too much performance penalty, though.